src/coreComponents/constitutive/solid/ElasticIsotropic.hpp

```diff
@@ -44,18 +44,21 @@ class ElasticIsotropicUpdates : public SolidBaseUpdates
    * @brief Constructor
    * @param[in] bulkModulus  The ArrayView holding the bulk modulus data for each element.
    * @param[in] shearModulus The ArrayView holding the shear modulus data for each element.
+   * @param[in] thermalExpansionCoefficient The ArrayView holding the thermal expansion coefficient data for each element.
    * @param[in] newStress    The ArrayView holding the new stress data for each quadrature point.
    * @param[in] oldStress    The ArrayView holding the old stress data for each quadrature point.
    * @param[in] disableInelasticity Flag to disable plasticity for inelastic models
    */
   ElasticIsotropicUpdates( arrayView1d< real64 const > const & bulkModulus,
                            arrayView1d< real64 const > const & shearModulus,
+                           arrayView1d< real64 const > const & thermalExpansionCoefficient,
```

Comment:
I double-checked in the other constitutive models. In `solid` and in most of the other constitutive models, the convention is to use `Coefficient` because we already have
- `biotCoefficient`
- `deltaCoefficient`
- `landCoefficient`
- `dilationCoefficient`
- `frictionCoefficient`
...

I always prefer when the variable names are spelled out completely, but we can change the convention if people are in favor of another convention.

One thing to note is that there is already a thermal expansion coefficient in the fluid model and it is spelled `thermalExpansionCoef` (see `ThermalCompressibleSinglePhaseFluid.hpp`). I would suggest to rename it to `thermalExpansionCoefficient` to be consistent.

Reply:
I am also in favor of spelling out variable names completely.